Fails to link using GCC 10.3.0

Found while building under Linux x86_64 using GCC 10.3.0.

<details><summary>Full build log</summary>

```
make config
make[1]: Entering directory '/usr/src/git/ctris'
./create_config.sh
make[1]: Leaving directory '/usr/src/git/ctris'
make build
make[1]: Entering directory '/usr/src/git/ctris'
gcc -Wall -fomit-frame-pointer -O3 -c ctris.c
ctris.c: In function 'main':
ctris.c:46:3: warning: 'strncpy' specified bound 40 equals destination size [-Wstringop-truncation]
   46 |   strncpy(default_name, getenv("USER"), 40);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
gcc -Wall -fomit-frame-pointer -O3 -c game.c
game.c: In function 'game_engine':
game.c:220:21: warning: this 'while' clause does not guard... [-Wmisleading-indentation]
  220 |                     while(old_get_key(board_win) != 'p');
      |                     ^~~~~
game.c:221:6: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the 'while'
  221 |      game_state = RUNNING_STATE;
      |      ^~~~~~~~~~
game.c:244:25: warning: this 'while' clause does not guard... [-Wmisleading-indentation]
  244 |                         while(old_get_key(board_win) != ' ');
      |                         ^~~~~
game.c:245:7: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the 'while'
  245 |       game_state = QUIT_STATE;
      |       ^~~~~~~~~~
gcc -Wall -fomit-frame-pointer -O3 -c screen.c
screen.c: In function 'add_user_to_highscore':
screen.c:342:2: warning: 'strncpy' specified bound 40 equals destination size [-Wstringop-truncation]
  342 |  strncpy(default_name, name, 40);
      |  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
gcc -Wall -fomit-frame-pointer -O3 -c brick.c
gcc -Wall -fomit-frame-pointer -O3 -c highscore.c
highscore.c: In function 'add_to_highscore':
highscore.c:80:2: warning: 'strncpy' specified bound 40 equals destination size [-Wstringop-truncation]
   80 |  strncpy(highscore.entry[9].name, name, 40);
      |  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
gcc -Wall -fomit-frame-pointer -O3 -o ctris ctris.o game.o screen.o brick.o highscore.o -lm -lncursesw
/usr/bin/ld: game.o:(.bss+0x1c3): multiple definition of `game_state'; ctris.o:(.bss+0x143): first defined here
/usr/bin/ld: game.o:(.bss+0x1c1): multiple definition of `counterclockwise_rotation'; ctris.o:(.bss+0x141): first defined here
/usr/bin/ld: game.o:(.bss+0x1c2): multiple definition of `old_style_keys'; ctris.o:(.bss+0x142): first defined here
/usr/bin/ld: game.o:(.bss+0x70): multiple definition of `rseed'; ctris.o:(.data+0x0): first defined here
/usr/bin/ld: game.o:(.bss+0x80): multiple definition of `default_name'; ctris.o:(.bss+0x0): first defined here
/usr/bin/ld: game.o:(.bss+0xc0): multiple definition of `highscore_file_path'; ctris.o:(.bss+0x40): first defined here
/usr/bin/ld: game.o:(.bss+0x1c0): multiple definition of `hide_cursor_workaround'; ctris.o:(.bss+0x140): first defined here
/usr/bin/ld: screen.o:(.bss+0x1e0): multiple definition of `hide_cursor_workaround'; ctris.o:(.bss+0x140): first defined here
/usr/bin/ld: screen.o:(.bss+0x20): multiple definition of `brick_digit'; game.o:(.bss+0x0): first defined here
/usr/bin/ld: screen.o:(.bss+0xa0): multiple definition of `default_name'; ctris.o:(.bss+0x0): first defined here
/usr/bin/ld: screen.o:(.bss+0x90): multiple definition of `rseed'; ctris.o:(.data+0x0): first defined here
/usr/bin/ld: screen.o:(.bss+0xe0): multiple definition of `highscore_file_path'; ctris.o:(.bss+0x40): first defined here
/usr/bin/ld: screen.o:(.bss+0x1e1): multiple definition of `counterclockwise_rotation'; ctris.o:(.bss+0x141): first defined here
/usr/bin/ld: screen.o:(.bss+0x1e2): multiple definition of `old_style_keys'; ctris.o:(.bss+0x142): first defined here
/usr/bin/ld: screen.o:(.bss+0x1e3): multiple definition of `game_state'; ctris.o:(.bss+0x143): first defined here
/usr/bin/ld: brick.o:(.data+0x0): multiple definition of `brick_digit'; game.o:(.bss+0x0): first defined here
/usr/bin/ld: brick.o:(.bss+0x0): multiple definition of `rseed'; ctris.o:(.data+0x0): first defined here
/usr/bin/ld: brick.o:(.bss+0x20): multiple definition of `default_name'; ctris.o:(.bss+0x0): first defined here
/usr/bin/ld: brick.o:(.bss+0x60): multiple definition of `highscore_file_path'; ctris.o:(.bss+0x40): first defined here
/usr/bin/ld: brick.o:(.bss+0x160): multiple definition of `hide_cursor_workaround'; ctris.o:(.bss+0x140): first defined here
/usr/bin/ld: brick.o:(.bss+0x161): multiple definition of `counterclockwise_rotation'; ctris.o:(.bss+0x141): first defined here
/usr/bin/ld: brick.o:(.bss+0x162): multiple definition of `old_style_keys'; ctris.o:(.bss+0x142): first defined here
/usr/bin/ld: brick.o:(.bss+0x163): multiple definition of `game_state'; ctris.o:(.bss+0x143): first defined here
/usr/bin/ld: highscore.o:(.bss+0x60): multiple definition of `highscore_file_path'; ctris.o:(.bss+0x40): first defined here
/usr/bin/ld: highscore.o:(.bss+0x0): multiple definition of `rseed'; ctris.o:(.data+0x0): first defined here
/usr/bin/ld: highscore.o:(.bss+0x20): multiple definition of `default_name'; ctris.o:(.bss+0x0): first defined here
/usr/bin/ld: highscore.o:(.bss+0x160): multiple definition of `hide_cursor_workaround'; ctris.o:(.bss+0x140): first defined here
/usr/bin/ld: highscore.o:(.bss+0x161): multiple definition of `counterclockwise_rotation'; ctris.o:(.bss+0x141): first defined here
/usr/bin/ld: highscore.o:(.bss+0x162): multiple definition of `old_style_keys'; ctris.o:(.bss+0x142): first defined here
/usr/bin/ld: highscore.o:(.bss+0x163): multiple definition of `game_state'; ctris.o:(.bss+0x143): first defined here
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:27: ctris] Error 1
make[1]: Leaving directory '/usr/src/git/ctris'
make: *** [Makefile:19: all] Error 2
```

</details>

--- a/brick.h
+++ b/brick.h
@@ -8,7 +8,7 @@ void find_index(char [4][4], unsigned char *, unsigned char *);
 char check_brick(char [BOARD_HEIGHT][BOARD_WIDTH], char [4][4], const unsigned char, const unsigned char);
 void draw_to_board(char [BOARD_HEIGHT][BOARD_WIDTH], char [4][4], const char, const unsigned char, const unsigned char);
 
-char brick_digit[7][4][4];
+extern char brick_digit[7][4][4];
 
 #endif
 

--- a/ctris.h
+++ b/ctris.h
@@ -44,9 +44,9 @@ void handle_quit_signal(const int);
 unsigned char get_rand(const unsigned char);
 int main(int, char *[]);
 
-char game_state, old_style_keys, counterclockwise_rotation, hide_cursor_workaround, highscore_file_path[256], default_name[40];
+extern char game_state, old_style_keys, counterclockwise_rotation, hide_cursor_workaround, highscore_file_path[256], default_name[40];
 
-unsigned long rseed;
+extern unsigned long rseed;
 
 #endif
 

